### PR TITLE
Add a description of how to develop to README

### PR DESCRIPTION
**Pull request by abcdefg-1234567**
Added explanation of how to develop red-dataset to README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,12 @@ end
 
 * [red-datasets-numo-narray](https://github.com/red-data-tools/red-datasets-numo-narray)
 
+## How to develop Red Datasets
+1. Fork https://github.com/red-data-tools/red-datasets 
+2. Create a feature branch from master
+3. Develop in the feature branch
+4. Pull request form the feature branch to https://github.com/red-data-tools/red-datasets
```
Comment (kou) [inline README.md:142]:
```suggestion
4. Pull request from the feature branch to https://github.com/red-data-tools/red-datasets
```

**Review by kou — APPROVED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (abcdefg-1234567) [reply README.md:142]: Thanks for pointing out the typo.

**Comment by kou** [comment]
Merged!

**Comment by abcdefg-1234567** [comment]
Thank you so much!

**Pull request merged**